Public/Src/Engine/Dll/Distribution/Grpc/GrpcConstants.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace BuildXL.Engine.Distribution.Grpc
+{
+    internal static class GrpcConstants
+    {
+        public const string TraceIdKey = "traceid-bin";

```

Comment:
>-bin [](start = 49, length = 4)

Why "-bin"?

Reply:
when you send binary data in the headers, you need to put '-bin' suffix to indicate that the value is binary. 

---
In reply to: [272744034](https://github.com/Microsoft/BuildXL/pull/91#discussion_r272744034) [](ancestors = 272744034)